package/src/components/Channel/Channel.tsx

```diff
@@ -948,8 +948,8 @@ const ChannelWithContext = <
           }
         }
         const hasLatestMessages = channel.state.latestMessages.length > 0;
-        channel.state.setIsUpToDate(hasLatestMessages);
-        setHasNoMoreRecentMessagesToLoad(hasLatestMessages);
+        channel.state.setIsUpToDate(!hasLatestMessages);
```

Comment:
Only mark everything as up to date once the "dummy" message set is added.

Reply:
This is still very wrong, but it seems that we have parts of the code that depend on this. Can look at a later iteration as to why this ever worked.